### ISSUE TITLE
edX Intro to Web A11y course page is missing a logo

### ISSUE THREAD

**Issue by wittjeff** [w3c/wai-website#1623]
### Resource URL

https://prod-edx-mktg-edit.edx.org/sites/default/files/logo-unesco-iite.png

### Description

Bug on course page: https://www.edx.org/learn/web-accessibility/the-world-wide-web-consortium-w3c-introduction-to-web-accessibility
Under More about this course, UNESCO iite logos are missing.
Path is https://prod-edx-mktg-edit.edx.org/sites/default/files/logo-unesco-iite.png
You seem to use that file in other locations (here) as well.
